.env.prod

```diff
@@ -7,6 +7,4 @@ FIRESTORE_PROJECT_ID = "find-a-doc-japan"
 # https://console.firebase.google.com/u/0/project/find-a-doc-japan/overview
 # DO NOT COMMIT THESE VALUES TO GIT!!!
 
-# FIRESTORE_SERVICE_ACCOUNT = 'enteryourkeyhere'
-
-
+# FIRESTORE_SERVICE_ACCOUNT = 'enteryourkeyhere'
```

Comment:
I think you can leave this out since it's already on main.

Reply:
@theyokohamalife I'll be sure to remove it (or figure out how to do that).

Reply:
@evan-desu  You should be able to run `git checkout main .env.prod` and then commit and push to have this removed.

Reply:
@theyokohamalife I'll try to do that later this evening. I gave up yesterday 😅